### ISSUE TITLE
support for cookie as jwt token storage

### ISSUE THREAD

**Issue by sjquant** [NovemberOscar/Sanic-JWT-Extended#13]
flask-jwt-extended supports using cookie as jwt token storage, has utility function like `set_access_cookie`, `set_refresh_cookie`, and config options like `JWT_COOKIE_CSRF_PROTECT`.

I'm developing website using Nuxt (Vue SSR Framework), and I want to use cookie for authentication in server-side rather than localStorage. But I don't think this project supports it now. Do you have any plan for this?

**Comment by rscarrera27** [comment]
Yes, cookie is currently not supported now. but I don't have detailed plan for that yet.

honestly, I have plan for 1.0 release with few major changes. I'll think about adding it to the 1.0 version, so I hope you wait.

**Comment by rscarrera27** [comment]
If you want, you can open cookie support PR

**Comment by sjquant** [comment]
@NovemberOscar Thanks for early response. I will try, but I'm not sure I can take much time!

**Comment by rscarrera27** [comment]
@sjquant `1.0.dev2` released and cookie will added soon. currently supports query and  headers.

**Comment by sjquant** [comment]
@NovemberOscar I'm really looking forward to that : )

**Comment by rscarrera27** [comment]
@sjquant 

Latest version `1.0.dev6` now contains support for reading cookie with CSRF support. 

cookie setter utils similar to `flask-jwt-extended` will be release in `1.0-dev.10` or later

**Comment by sjquant** [comment]
@NovemberOscar Thanks for actively maintaining this project.
